llama2.mojo

```diff
@@ -514,7 +514,7 @@ fn accum(inout a: TensorF32, b: TensorF32) -> None:
 
     @parameter
     fn _acc[_nelts: Int](j: Int):
-        a.simd_store[_nelts](j, a.simd_load[_nelts](j) + b.simd_load[_nelts](j))
+        a.load[width=_nelts](j, a.load[width=_nelts](j) + b.load[width=_nelts](j))
```

Comment:
this first instance of `a.load` needs to be `a.store`.  We are updating  the value at index `j`.